include/seqan3/io/sequence/sequence_file_format.hpp

```diff
@@ -66,10 +99,25 @@ concept bool sequence_file_format_concept = requires (t v)
     };
 
 };
+//!\endcond
 
 namespace detail
 {
 
+/*!
+ * \brief Check whether all given formats meet the sequence_file_format_concept.
```

Comment:
This would be more beautiful and faster if you would use
```c++
template<typename variant_type, std::size_t... I>
constexpr bool meets_concept_sequence_file_format_impl(std::index_sequence<I...>)
{
    return (sequence_file_format_concept<std::variant_alternative_t<I, variant_type>> && ... true);
}

template<typename variant_type>
constexpr bool meets_concept_sequence_file_format()
{
    using index_sequence_t = std::make_integer_sequence<std::variant_size_v<variant_type>>;
    return meets_concept_sequence_file_format_impl<variant_type>(index_sequence_t{});
}
```